sources2csr/ngs_txt_reader.py

```diff
@@ -25,7 +27,7 @@ def read_data(self, filename: str) -> Optional[Sequence[NGS]]:
         if data:
             sample_id_col_num = 0
             for col_value in data[0]:
-                if col_value.startswith('PMC'):
+                if col_value not in ['Gene Symbol', 'Gene ID', 'Locus ID', 'Cytoband']:
```

Comment:
or maybe case insensitive?

Reply:
That is a tricky one, in [csr2cbioportal.py](https://github.com/thehyve/python_csr2transmart/blob/master/csr2cbioportal/csr2cbioportal.py#L161), the column names are referred to in a case-sensitive manner. We could convert all column headers to either lowercase or uppercase, but then also the sample identifiers are converted.

Reply:
ah ok, true. I didn't notice the renaming on the later stage